src/Components/Dropdowns/Composed/TypeAheadDropDown.tsx

```diff
@@ -94,28 +66,41 @@ export const TypeAheadDropDown: FC<OwnProps> = ({
 
   const [selectIndex, setSelectIndex] = useState(-1)
   const [queryResults, setQueryResults] = useState(items)
-  const [menuOpen, setMenuOpen] = useState<MenuStatus>(MenuStatus.Closed)
-
+  const [menuStatus, setMenuStatus] = useState<MenuStatus>(MenuStatus.Closed)
+  const [userHasTyped, setUserHasTyped] = useState(false)
```

Comment:
this keeps track of whether the user has typed or not, this helps us display a full list of items when viewing the dropdown. Instead of this: 


We see the full list : 
<img width="262" alt="Screen Shot 2022-08-01 at 1 47 40 PM" src="https://user-images.githubusercontent.com/18511823/182244016-695e7a7a-faf2-4a24-b99e-f66cf862aca1.png">

Reply:
I didn't understand this one: in the second image, why are options other than the options containing the text 'Los Angel' shown?

Reply:
There are two scenarios to consider here:

When the user hasn't typed anything (input has whatever previous value it had), when they click the dropdown in that state, we want to show the entire list and not filter it by the typed value. 

When the user starts typing, we want to then start filtering.

Reply:
The second image is the user just clicking on the dropdown, and has not begun typing anything in it yet.